packages/server/dataloader/customLoaderMakers.ts

```diff
@@ -147,13 +160,33 @@ export const userTasks = (parent: RethinkDataLoader) => {
 
       const entryArray = await Promise.all(
         uniqKeys.map(async (key) => {
-          const {first, after, userIds, teamIds, archived, includeUnassigned} = key
+          const {
+            first,
+            after,
+            userIds,
+            teamIds,
+            archived,
+            statusFilters,
+            filterQuery,
+            includeUnassigned
+          } = key
           const dbAfter = after ? new Date(after) : r.maxval
 
           let teamTaskPartial = r.table('Task').getAll(r.args(teamIds), {index: 'teamId'})
-          if (userIds) {
+          if (isNonEmptyArray(userIds)) {
             teamTaskPartial = teamTaskPartial.filter((row) => r(userIds).contains(row('userId')))
           }
+          if (isNonEmptyArray(statusFilters)) {
+            teamTaskPartial = teamTaskPartial.filter((row) =>
+              r(statusFilters).contains(row('status'))
+            )
+          }
+          if (filterQuery) {
+            // TODO: deal with tags like #archived and #private. should strip out of plaintextContent??
+            teamTaskPartial = teamTaskPartial.filter(
```

Comment:
on second thought, this todo might be a moot point since we don't want to show these as available for scoping anyway.